src/string.cr

```diff
@@ -1488,6 +1512,15 @@ class String
     end
   end
 
+  # Returns the index of `c` in the string, or `nil` if the string is not present.
```

Comment:
I agree, I wasn't sure how much I should mess with existing code! Here's the refactor:

https://github.com/manastech/crystal/commit/29cab3169ad613baba601d5276e910dac69bb62f